package.json

```diff
@@ -1,60 +1,90 @@
 {
   "name": "sanity-plugin-fieldwork",
-  "version": "1.1.2",
+  "version": "1.0.0",
```

Comment:
I'm guessing we need to keep going with the old version list? (eg maybe this is v2?)

Reply:
Or actually, I guess if you merge this major this will happen automatically? :D